Core/src/main/java/org/tribuo/ensemble/WeightedEnsembleModel.java

```diff
@@ -45,16 +48,35 @@
 
     protected final EnsembleCombiner<T> combiner;
 
-    public WeightedEnsembleModel(String name, EnsembleModelProvenance description, ImmutableFeatureMap featureIDMap,
+    /**
+     * Constructs an ensemble model which uses uniform weights.
```

Comment:
Since user's aren't generally going to be getting `WeightedEnsembleModel`s through this constructor, we should state that.

```suggestion
     * Unless you are implementing a {@link org.tribuo.Trainer} you should
     * not use this constructor directly. Instead use {@link #createEnsembleFromExistingModels(String, List<Model<T>>, EnsembleCombiner<T>)}.
     * <p>
     * Constructs an ensemble model which uses uniform weights.
```